apk-generator/v2/app/static/js/main.js

```diff
@@ -90,6 +92,16 @@ $jsonUploadInput.change(function () {
     }
 });
 
+$authOptionCheckbox.change(
+    function () {
+        if (this.checked) {
+            isAuthEnabled = true;
+        } else {
+            isAuthEnabled = false;
+        }
```

Comment:
This can be just reduced to `$authOptionCheckbox.checked` at the time of form submit. No need to create another variable to track exact state of the switch